Cargo.toml

```diff
@@ -38,7 +39,7 @@ serde_codegen = { version = "0.8", optional = true }
 # Note that not all features are available on all platforms, so even though a
 # feature is enabled some other feature may be used instead.
 [features]
-default = ["libunwind", "libbacktrace", "dladdr", "dbghelp"]
+default = ["libunwind", "libbacktrace", "coresymbolication", "dladdr", "dbghelp"]
```

Comment:
Ok, this paired with a lot of usage on google seems to point to this being a reasonable decision. Sounds good to me!